express-with-nodemon-browsersync/README.md

```diff
@@ -0,0 +1,34 @@
+# Express with Nodemon & Browsersync
+
+## Functionality
+
+Auto restart [ExpressJS](http://expressjs.com/) server and auto reload browser on file changes.
+
+This setup uses file watchers when the server is started in development mode (see [usage](#usage)).
+If server files change (e.g. [`index.js`](index.js)), the server itself automatically restarts (using [Nodemon](http://nodemon.io/)).
+If browser files change (e.g. [`src/index.html`](src/index.html) or [`src/index.css`](src/index.html)) the browser automatically reloads (or injects) (using [Browsersync](https://www.browsersync.io/)).
+Interactions are also synced when multiple browser windows are connected.
+
+Note: Automatic browser reloading only happens on the proxy `http://localhost:9778`, not the regular `http://localhost:9777`.
+
+## Usage
+
+After installing dependencies using `npm install` the following scripts are available:
+
+`npm run ...` | Description
+---|---
+`start` | Alias for `start:dev`.
+`start:dev` | Starts a development server on `http://localhost:9777` ("xprs" in T9) and `http://localhost:9778` with live reload and synced interaction.
```

Comment:
[Typing text on a numeric keypad](https://en.wikipedia.org/wiki/T9_(predictive_text)). So for typing "xprs" you would use the numbers 9 7 7 7. It's just a trick I use to make sure I have a different port number for every project.